sirepo/uri.py

```diff
@@ -80,6 +81,10 @@ def local_route(sim_type, route_name=None, params=None, query=None):
     return app_root(t) + '#' + u + _query(query)
 
 
+def root_redirect(path):
+    return simulation_db.SCHEMA_COMMON.rootRedirectUri.get(path)
```

Comment:
Document what this is, since other methods are documented. I would call the arg "uri", not "path". "path" implies file system value. "uri" in this case is a part of a uri.

Reply:
https://github.com/radiasoft/sirepo/pull/2786/commits/1912882dc322db1f33962e87148aeddc9c6ef2d7